docs/authentication.md

```diff
@@ -55,7 +55,8 @@ spec:
     rbac: {}
 ```
 
-Once the cluster is up, or after you've performed a rolling update to an existing cluster with `kops rolling-update cluster ${CLUSTER_NAME} --instance-group-roles=Master --force --yes`, you will need to create the AWS IAM authenticator
+Once done you need to run `kops update cluster ${CLUSTER_NAME} --yes; kops rolling-update cluster ${CLUSTER_NAME} --instance-group-roles=Master --cloudonly --force --yes`.
```

Comment:
That's a pretty aggressive rolling-update - do we know why it is needed?  I guess it's because we haven't created the configmap yet?

Reply:
Yeah, even if the configmap is created once the first master rolls the aws-iam-authenticator daemonset is added to all masters and while they other masters do have the configmap, they do not have configuration set so the daemon cannot work.